aiohttp_sse/__init__.py

```diff
@@ -165,3 +168,28 @@ def _ping(self):
         while True:
             yield from asyncio.sleep(self._ping_interval, loop=self._loop)
             self.write(b': ping\r\n\r\n')
+
+
+class EventSourceResponseCM(EventSourceResponse):
+    def __init__(self, request, *, status=200, reason=None, headers=None):
+        super().__init__(status=status, reason=reason, headers=headers)
+        self._request = request
+
+    @asyncio.coroutine
+    def __aenter__(self):
+        self.start(self._request)
```

Comment:
just noticed in aiohttp docs,  `start(request)` is deprecated, we should use `prepare` instead